point_cloud_preprocessing/CMakeLists.txt

```diff
@@ -0,0 +1,94 @@
+cmake_minimum_required(VERSION 3.5)
+project(point_cloud_preprocessing)
+
+set(CMAKE_CXX_STANDARD 14)
+add_compile_options(-Wall -Wextra -Wpedantic)
+
+
+set(CMAKE_EXPORT_COMPILE_COMMANDS ON)
+
+
+
+set(SRC_FILES
+  src/PointCloudPreprocessor.cpp
+  src/Parameters.cpp
+)
+
+find_package(catkin REQUIRED
+  COMPONENTS
+    ${CATKIN_REQUIRED_COMPONENTS}
+)
+
+
+find_package(PCL 1.10 REQUIRED)
+
+find_package(OpenMP QUIET)
+if (OpenMP_FOUND)
+  add_compile_options("${OpenMP_CXX_FLAGS}")
+  add_definitions(-DGROUND_PLANE_REMOVAL_OPENMP_FOUND=${OpenMP_FOUND})
```

Comment:
Careful, you should rename this to -DPOINTCLOUD_PREPROCESSING_OPENMP_FOUND

Reply:
Can I not just remove all the OpenMP stuff in the CMakeLists.txt. It is not just in this package, I just realize or is it related to pcl

Reply:
I removed the OpenMP stuff. it was still compiling therefore I think it should be fine